examples/todoapp/buildSrc/buildSrc/README.md

```diff
@@ -0,0 +1,4 @@
+# Why this subfolder/submodule
+
+This folder, `buildSrc/buildSrc`, force compiling `Deps.kt` file, so it can be used
```

Comment:
So the Deps.kt file is moved from `/buildSrc/buildSrc` to `/buildSrc` folder, but we still need this empty folder?

Reply:
It is not empty, it has two files, `README.md` and `build.gradle.kts` indicating where `Deps.kt` file is.

Reply:
So if we still need the nested `buildSrc` folder, I would keep Deps.kt in there, because there is less configuration code and it is also less confusing. Other changes LGTM.

Reply:
Changed!